### PR TITLE
first pass

### PR DESCRIPTION
**Pull request by willtp87**
requires
https://github.com/discoverygarden/islandora_rest/pull/7
https://github.com/discoverygarden/islandora_solution_pack_manuscript/pull/6
https://github.com/Islandora/islandora_openseadragon/pull/37

### CODE REVIEW

File: theme/theme.inc [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+ * Implements hook_preprocess_theme().
+ */
+function islandora_paged_tei_seadragon_preprocess_islandora_paged_tei_seadragon_viewer(array &$variables) {
+  module_load_include('inc', 'islandora_paged_tei_seadragon', 'includes/utilities');
+  module_load_include('inc', 'islandora', 'includes/authtokens');
+  $viewer_module_path = drupal_get_path('module', 'islandora_paged_tei_seadragon');
+  $manuscript = $variables['object'];
+  $transform = $variables['transform_object'];
+  if (isset($_GET['islandora_paged_content_page'])) {
+    $page_pid = $pages[$_GET['islandora_paged_content_page']]['PID'];
```
Comment (jordandukart) [inline theme/theme.inc]: `$pages` not defined think it's supposed to be `$variables['pages']`?

File: theme/theme.inc [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+/**
+ * Implements hook_preprocess_theme().
+ */
+function islandora_paged_tei_seadragon_preprocess_islandora_paged_tei_seadragon_viewer(array &$variables) {
+  module_load_include('inc', 'islandora_paged_tei_seadragon', 'includes/utilities');
+  module_load_include('inc', 'islandora', 'includes/authtokens');
+  $viewer_module_path = drupal_get_path('module', 'islandora_paged_tei_seadragon');
+  $manuscript = $variables['object'];
+  $transform = $variables['transform_object'];
+  if (isset($_GET['islandora_paged_content_page'])) {
```
Comment (jordandukart) [inline theme/theme.inc:16]: Should probably be looking to access `$_GET['islandora_paged_content_page']['PID']`?

Comment (willtp87) [inline theme/theme.inc:16]: I had misspoke.

File: islandora_paged_tei_seadragon.module [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+ */
+
+/**
+ * Implements hook_islandora_viewer_info().
+ */
+function islandora_paged_tei_seadragon_islandora_viewer_info() {
+  return array(
+    'islandora_paged_tei_seadragon' => array(
+      'label' => t('Paged Content TEI OpenSeadragon viewer'),
+      'description' => t('Two pained viewer with rendered TEI on one side and
```
Comment (jordandukart) [inline islandora_paged_tei_seadragon.module:15]: Very painful.

**Pull request merged**